docs/src/lib/indexing.md

```diff
@@ -42,22 +42,23 @@ The rules for a valid type of index into a column are the following:
       selected.
     * a `Between` expression (see
       [DataAPI.jl](https://github.com/JuliaData/DataAPI.jl));
-      `Between(first, last)` selects the columns between `first` and `last`;
+      `Between(first, last)` selects the columns between `first` and `last` inclusively;
     * an `All` expression (see
       [DataAPI.jl](https://github.com/JuliaData/DataAPI.jl)); `All()` selects
       all columns, equivalent to `:`;
-    * a colon literal `:`.
+    * a literal colon `:` (selects all columns).
 
-The rules for a valid type of index into a row are the following:
+The following types are a valid row index:
```

Comment:
Maybe delete the word "types" since I'm not sure `:` and `!` count as types. I also considered "The rules for a valid row index are the following:" instead.

Reply:
we index with values not types.

Reply:
Is a vector a value? I just deleted both "types" and "values". I don't think either word helps clarify. The list contains mostly types, but I think it is clear that `row[Int32]` wouldn't make any sense.

Reply:
vector is a value. Types are e.g. `Int64` or `Vector{Int}`, but you are not allowed to use them for indexing.

Reply:
Okay, I put "value" back in line 51, but then "value" in line 52 must change. I used "scalar" in line 52 in place of "value". I'm not sure that this is any clearer, but I'll let you choose your preference: see commit 29c2412.